src/services/products.ts

```diff
@@ -1,9 +1,33 @@
 import { Products } from '../models/Products';
+import { filterProducts } from '../utils/filterProdacts';
+import { getProductsWitUrl } from '../utils/getProductsWitUrl';
+import { ParsedQs } from 'qs';
 
-function getAll() {
-  return Products.findAll();
-}
+const getProductsWithPagination = async (pageNumber: number, limitNumber: number, sort: string | ParsedQs | string[] | ParsedQs[]) => {
```

Comment:
Let's move `sort: string | ParsedQs | string[] | ParsedQs[]` type to a separate file.